src/index.tsx

```diff
@@ -481,7 +482,4 @@ export {
   useWallet,
   getProviderString,
   getProviderFromUseWalletId,
-  KNOWN_CHAINS,
 }
```

Comment:
KNOWN_CHAINS is no longer available from use-wallet now?  This will be a breaking change for apps that depend on it like Aragon Client, Wrapper, etc.  Make sure this get released as major release.